pathd/path_pcep_cli.c

```diff
@@ -1018,10 +1018,14 @@ static int path_pcep_cli_pcc_delete(struct vty *vty)
 }
 
 static int path_pcep_cli_pcc_pcc_msd(struct vty *vty, const char *msd_str,
-				     long msd)
+				     long msd, bool reset)
```

Comment:
can we name this something besides reset?  reset has implications of restarting instead of turning on / off.  `bool on` or `bool turn_on`